docs/spec_draft.md

```diff
@@ -1247,6 +1248,39 @@ defined and one of the following:
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.is_finite
+
+### Semantics
+
+Performs element-wise check whether the value in `x` is finite (i.e. is neither
++inf, -inf, nor NaN) and produces a `y` tensor.
+
+### Inputs
+
+| Name | Type                          |
+|------|-------------------------------|
+| `x`  | tensor of floating-point type |
```

Comment:
can we open a ticket to rename the aliases to operand and result in the ODS

Reply:
There is already a ticket for that #351. Updated to include this op.